spec/pom.xml

```diff
@@ -58,8 +58,8 @@
                         <configuration>
                             <rules>
                                 <requireJavaVersion>
-                                    <version>[1.8.0,1.9.0)</version>
-                                    <message>You need JDK8 or lower</message>
+                                    <version>[1.8.0,)</version>
+                                    <message>You need JDK8 or higher</message>
                                 </requireJavaVersion>
```

Comment:
I don't think this should be part of the package rename.

Reply:
Unfortunately, I had to update the copyright and spec plugins to allow this to work.
Upgrading those plugins triggered this enforcer rule.
So I adjusted the enforcer rule to be consistent to how the other projects behave.

But I agree, it feels way out of place in this PR.
I can make that change a different PR, and then I can modify this PR to not include this change, but that means this PR becomes dependent on another PR (which is something I don't like doing)
You OK with this approach?

Reply:
Fair enough. Lets just include it in this PR. Maybe include a note in the commit message?